triage/issue-tracker.rst

```diff
@@ -84,16 +84,43 @@ The submission form has only two fields that you need to fill:
   information as appropriate). In particular, *what version of Python* you
   were using.
 
-Understanding the issue's progress and status
----------------------------------------------
+It is also acceptable to tag someone to if you think the issue
+should be brought to their attention.  Use the :ref:`experts` to know
+who wants to be added to the nosy list for issues targeting specific areas.
```

Comment:
After this was moved here, the "It is also" is now out of context/out of place here, and it is also now lacking a mention about _how_ to actually tag people on GitHub, which is particularly important given the audience here may include people relatively new to GitHub who might not be as familiar with this. Therefore, I suggest a few adjustments accordingly:

```suggestion
You can tag someone, with :samp:`@{username}` in a comment,
if you think the issue should be brought to their attention.
Use the :ref:`experts` to know who wants to be tagged for specific areas.
```

Also, I eliminated some now-outdated text about "added to the nosy list"